internal/cmd/export.go

```diff
@@ -484,6 +490,40 @@ func gatherPGOCLIVersion(_ context.Context,
 	return nil
 }
 
+func gatherPostgresClusterNames(clusterName string, ctx context.Context, cmd *cobra.Command, tw *tar.Writer, client dynamic.NamespaceableResourceInterface) error {
+	result, err := client.List(ctx, metav1.ListOptions{})
+
+	if err != nil {
+		return err
+	}
+
+	data := []byte{}
+	for _, item := range result.Items {
+		ns, found, err := unstructured.NestedString(item.Object, "metadata", "namespace")
+		if !found {
+			return fmt.Errorf("key not found: metadata.namespace")
+		}
+		if err != nil {
+			return err
+		}
+		name, found, err := unstructured.NestedString(item.Object, "metadata", "name")
+		if !found {
+			return fmt.Errorf("key not found: metadata.name")
+		}
+		if err != nil {
+			return err
+		}
```

Comment:
🌱 Every PostgresCluster returned from the API will have a namespace and name. There's also no harm in proceeding if we _do_ encounter something unexpected. I think we prefer to gather the remainder of the list rather than give up early.

```suggestion
		ns, _, _ := unstructured.NestedString(item.Object, "metadata", "namespace")
		name, _, _ := unstructured.NestedString(item.Object, "metadata", "name")
```